test/stats.js

```diff
@@ -111,6 +113,70 @@ suite('Stats', function () {
       });
     }); // end getting stats
 
+    test('Getting cached stats for a survey', function () {
```

Comment:
Would it make sense to split this into three scenarios? 
- Getting cached stats
- Getting cached stats (new entries)
- Getting cached stats (deleted entries)